src/content/news/2021-03-23-real-estate-tech-seminar.md

```
@@ -6,26 +6,45 @@ category: "お知らせ"
-東京海上日動火災保険株式会社　本店営業第三部　清水 隆
+
+####登壇者（敬称略）
+
+-   株式会社Geolonia　代表取締役CEO　宮内 隆行
+-   理事　滝沢 潔 　（株式会社ライナフ　代表取締役）　モデレーター
+-   理事　巻口 成憲　（リーウェイズ株式会社　代表取締役）
+-   理事　岡村 雅信　（株式会社UPDATA　代表取締役）
+-   理事　名村 晋治　（株式会社サービシンク　代表取締役）
+-   理事　落合 孝文　（渥美坂井法律事務所 弁護士）
+
+####登壇内容・タイムテーブル
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)